docs/reference/ml/anomaly-detection/ml-configuring-alerts.asciidoc

```diff
@@ -0,0 +1,85 @@
+[role="xpack"]
+[[ml-configuring-alerts]]
+= Configuring {anomaly-detect} alerts
+
+{anomaly-detect-cap} alerts run scheduled checks on an {anomaly-job} or a group 
+of jobs to detect anomalies with certain conditions. If an {anomaly} meets the 
+conditions, the alert triggers the defined action. For example, you can create 
+an alert that checks an {anomaly-job} in every fifteen minutes for critical 
+anomalies and notify you in an email. This page helps you to configure an 
+{anomaly-detect} alert. To learn more about alerts in the {stack}, refer to 
+{kibana-ref}/alerting-getting-started.html#alerting-getting-started[Alerting and Actions].
+
+
+[[creating-anomaly-alerts]]
+== Creating an alert
+
+{anomaly-detect-cap} alerts can be created in the {anomaly-job} wizard, from the 
+job list or under **Stack management > Alerts and Actions**. On the *Create 
+alert* template, select *{anomaly-detect-cap} alert* under the {ml-cap} section, 
+then give a name to the alert and optionally provide tags.
+
+[role="screenshot"]
+image::images/ml-anomaly-alert-type.jpg["Creating an anomaly detection alert"]
+
+Specify the time interval for the alert to check detected anomalies. It is 
+recommended to select an interval that is similar to the bucket span of the 
+associated job. You can also select a notification option by using the _Notify_ 
+selector. There are three notification options:
+
+* Notify only on status change: actions are triggered when the alert status 
+  changes.
+* Notify every time alert is active: actions are triggered every time when the 
+  alert checks changes.
+* Notify on a custom action interval: actions run by using an interval that you 
+  configured.
+  
+Select the {anomaly-job} or the group of {anomaly-jobs} that is checked by the 
+alert. If you assign additional jobs to the group, the alert automatically 
+checks the new jobs the next time when it runs.
+
+You can select the result type of the {anomaly-job} that triggers the alert. 
+{anomaly-detect-cap} produces three types of results: bucket, record, and 
+influencer. Bucket result type is selected by default.
```

Comment:
If you accept my suggestion on the first half of the sentence, then this should change too. I also think the default selection detail is unnecessary:
```suggestion
influencer results.
```